weblate/addons/models.py

```diff
@@ -183,6 +184,17 @@ def unit_pre_create_handler(sender, unit, **kwargs):
         addon.addon.unit_pre_create(unit)
 
 
+@receiver(post_save)
```

Comment:
You should filter here to avoid signal trigger for all models, see https://docs.djangoproject.com/en/2.0/topics/signals/#connecting-to-signals-sent-by-specific-senders, then the `if sender...` block is not needed.

Reply:
You're right. That looks cleaner. TIL